src/card/Card.tsx

```diff
@@ -175,6 +175,7 @@ const NeoCard = ({
             height={report.height}
             heightPx={height}
             fields={report.fields}
+            fieldsAll={report.fieldsAll}
```

Comment:
we can replace `fieldsAll` by `schema` everywhere, also replace all getters, settings, and actions.

Reply:
Done